Resources/doc/reference/routing.rst

```diff
@@ -107,6 +107,25 @@ explicitly this defaults to the action name.
         }
     }
 
+Make use of all route parameters
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+As the ``add`` method create a Symfony ``Route``, you can use all constructor parameters of the ``Route`` as parameters
```

Comment:
"you can use all constructor **arguments**" would be better and will avoid repetition, isn't it?